Editor/EditorCore/MaterialEditor.cs

```diff
@@ -15,6 +15,13 @@ namespace UnityEditor.ProBuilder
     /// </summary>
     sealed class MaterialEditor : ConfigurableWindow
     {
+        class MaterialShortcutContext : IShortcutContext
+        {
+            public bool active
+                => EditorWindow.focusedWindow is SceneView
+                   && instance != null;
```

Comment:
this should also check for this:
MeshSelection.selectedObjectCount > 0 && instance.m_QueuedMaterial.value != null;

For consistency with the Validation function and to avoid nullref :D